### PR TITLE
use 'ping -6' when ping6 is not available

### PR DESCRIPTION
**Pull request by l-hellmann**

### CODE REVIEW

File: src/nettools/ping.go [hunk 1]
```diff
@@ -2,20 +2,73 @@ package nettools
...
 	"runtime"
+	"strings"
 )
 
 func HasIPv6PingCommand() bool {
+	if hasPing6Command() {
+		return true
+	}
+	_, err := exec.LookPath("ping")
+	if err == nil {
```
Comment (MichalSalon) [inline src/nettools/ping.go:16]:
```suggestion
	if _, err := exec.LookPath("ping"); err == nil {
```

File: src/vpn/dnsIsAlive.go [hunk 2]
```diff
@@ -13,11 +13,12 @@ func (h *Handler) dnsIsAlive() (bool, error) {
 	ctx, cancel := context.WithTimeout(context.Background(), time.Second*30)
 	defer cancel()
 
-	if !nettools.HasIPv6PingCommand() {
+	if nettools.HasIPv6PingCommand() {
```
Comment (MichalSalon) [inline src/vpn/dnsIsAlive.go:16]: I don't think the removal of `!` is correct.

**Review by MichalSalon — CHANGES_REQUESTED** [review]
Add a changelog

```
## [v0.12.17] - 2023-01-09

### Changed
- use `ping -6` when `ping6` is not available
```

File: src/vpn/dnsIsAlive.go (hunk 2 above)
Comment (l-hellmann) [reply src/vpn/dnsIsAlive.go]: Yeah, my bad

**Review by MichalSalon — APPROVED** [review]

**Pull request merged**